frontend/src/features/vessel_sidebar/beacon_malfunctions/resume/YearBeaconMalfunctions.js

```diff
@@ -177,7 +177,7 @@ const Text = styled.div`
   font-size: 13px;
   font-weight: 500;
   width: 95%;
-  ${props => props.isEmpty ? null : 'cursor: pointer;'} 
+  ${props => props.isEmpty ? null : 'cursor: pointer;'}
```

Comment:
```suggestion
  pointer: ${props => props.isEmpty ? 'default' : 'pointer'};
```

Reply:
When we can use a default value, I find it more readable and it avoids using null.

When we have no choice and must "create" multiple props conditionally, we can use [mixins](https://styled-components.com/docs/api#css) via `css`. We can even do that within the code:

```tsx
styled.div<{
  isHoverable: boolean
}>`
  color: blue;

  ${p => p.isHoverable && css`
     :hover {
       color: yellow;
       text-decoration: underline;
     }
  `}
`
```

, but it's often cleaner with separate mixins for complex cases.

Reply:
FYI this is old code and I didn't switched the file to .ts, but yep go refactor it like this!